Gordon360/Models/ViewModels/RecIM/ActivityViewModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Gordon360.Models.CCT;
+using System;
+using System.Collections.Generic;
+
+namespace Gordon360.Models.ViewModels.RecIM
+{
+    public class ActivityViewModel
+    {
+        public int ID { get; set; }
+        public string Name { get; set; }
+        public DateTime RegistrationStart { get; set; }
+        public DateTime RegistrationEnd { get; set; }
+        public Sport Sport { get; set; }
+        public string Status { get; set; }
+        public int? MinCapacity { get; set; }
+        public int? MaxCapacity { get; set; }
+        public bool SoloRegistration { get; set; }
+        public string Logo { get; set; }
+        public bool Completed { get; set; }
+
+        public virtual IEnumerable<SeriesViewModel> Series { get; set; }
+        public virtual IEnumerable<TeamViewModel> Team { get; set; }
```

Comment:
What was your goal in making these properties `virtual`? Do you plan to implement them differently in a derived class? If not, you don't need to mark them as virtual.

Reply:
I think I had them copied from the models, so I'll remove them. I honestly did not even notice the virtual tag.